tests/unit/slidingPanel/__snapshots__/slidingPanel.spec.js.snap

```diff
@@ -102,6 +102,34 @@ exports[`SlidingPanel #render() closes the panel when active prop changes to fal
 </div>
 `;
 
+exports[`SlidingPanel #render() do not render footer if prop is not passed or falsy 1`] = `
+<SlidingPanel
+  closeOnOverlayClick={true}
+  direction="right"
+  subheader={null}
+  useDefaultLeftBlock={false}
+  width="480px"
+>
+  <div
+    className="ui-sliding-panel-overlay ui-sliding-panel-overlay_hidden"
+    onClick={[Function]}
+  >
+    <div
+      className="ui-sliding-panel ui-sliding-panel_right"
+      style={
+        Object {
+          "width": "480px",
```

Comment:
can you pls also add the variations of the sliding panel for small and large version and not use the fixed inline styles?

Reply:
Good point. Let's discuss that on Monday how to implement that in better way.

Reply:
So, let's keep it for now and refactor in separate PR. Issue is here -> #240